From 97d99be6e911146cc09d6080b71c3495b713e8eb Mon Sep 17 00:00:00 2001
From: Victor Kaplansky <victork@redhat.com>
Date: Mon, 14 Mar 2016 14:48:52 +0100
Subject: [PATCH 2/3] tests: vhost-user: disable unless CONFIG_VHOST_NET

RH-Author: Victor Kaplansky <victork@redhat.com>
Message-id: <1457966268-6229-2-git-send-email-victork@redhat.com>
Patchwork-id: 69783
O-Subject: [RHEV-7.3/7.2.z qemu-kvm-rhev PATCH 1/2] tests: vhost-user: disable unless CONFIG_VHOST_NET
Bugzilla: 1316156
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Acked-by: Gal Hammer <ghammer@redhat.com>
RH-Acked-by: Marcel Apfelbaum <marcel@redhat.com>

From: "Michael S. Tsirkin" <mst@redhat.com>

vhost-user depends on vhost-net. We should probably fix that.
For now, let's disable the test otherwise.

Signed-off-by: Victor Kaplansky <victork@redhat.com>
Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
Signed-off-by: Peter Maydell <peter.maydell@linaro.org>
(cherry picked from commit 7fe34ca9c2e99560dc65395d599a6920624b127d)
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 tests/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/Makefile b/tests/Makefile
index 8d7522e..db7eb31 100644
--- a/tests/Makefile
+++ b/tests/Makefile
@@ -158,7 +158,9 @@ gcov-files-i386-y += hw/usb/dev-storage.c
 check-qtest-i386-y += tests/usb-hcd-xhci-test$(EXESUF)
 gcov-files-i386-y += hw/usb/hcd-xhci.c
 check-qtest-i386-y += tests/pc-cpu-test$(EXESUF)
+ifeq ($(CONFIG_VHOST_NET),y)
 check-qtest-i386-$(CONFIG_LINUX) += tests/vhost-user-test$(EXESUF)
+endif
 check-qtest-x86_64-y = $(check-qtest-i386-y)
 gcov-files-i386-y += i386-softmmu/hw/timer/mc146818rtc.c
 gcov-files-x86_64-y = $(subst i386-softmmu/,x86_64-softmmu/,$(gcov-files-i386-y))
-- 
1.8.3.1

